services/ui-src/src/shared/commonQuestions/MeasureStratification/NDR/ndrFields.tsx

```diff
@@ -152,12 +152,15 @@ const buildNDRComponent = (
       >
         Please review the auto-calculated rate and revise if needed.
       </CUI.Heading>
-      {sets.map((_label, idx) => (
-        <CUI.Box mt="1rem">
-          <CUI.Heading fontSize="16px">{_label.label}</CUI.Heading>
-          {rateArrays[idx]}
-        </CUI.Box>
-      ))}
+      {sets.map(
+        (_label, idx) =>
+          rateArrays[idx].length > 0 && (
+            <CUI.Box mt="1rem">
+              <CUI.Heading fontSize="16px">{_label.label}</CUI.Heading>
+              {rateArrays[idx]}
+            </CUI.Box>
+          )
+      )}
```

Comment:
Usually an underscore prefix indicates an unused variable. Since `_label` is used, I'd suggest removing that prefix.

```suggestion
      {sets.map(
        (label, idx) =>
          rateArrays[idx].length > 0 && (
            <CUI.Box mt="1rem">
              <CUI.Heading fontSize="16px">{label.label}</CUI.Heading>
              {rateArrays[idx]}
            </CUI.Box>
          )
      )}
```

Reply:
fixed, also changed it to set.label instead of label.label 😰